modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/episode/EpisodeContainerFragment.kt

```diff
@@ -186,14 +182,6 @@ class EpisodeContainerFragment :
             state = BottomSheetBehavior.STATE_EXPANDED
             skipCollapsed = true
         }
-        // Ensure the dialog ends up the full height of the screen
-        // Bottom sheet dialogs get wrapped in a sheet that is WRAP_CONTENT so setting MATCH_PARENT on our
-        // root view is ignored.
-        bottomSheetDialog?.setOnShowListener {
-            view.updateLayoutParams<ViewGroup.LayoutParams> {
-                height = Resources.getSystem().displayMetrics.heightPixels
-            }
-        }
```

Comment:
This wasn't giving the view the correct height on different versions of Android, and the workaround of having the `MatchParentFrameLayout` component fixed the bottom padding issues.